src/inc/readytorun.h

```diff
@@ -15,7 +15,7 @@
 
 #define READYTORUN_SIGNATURE 0x00525452 // 'RTR'
 
-#define READYTORUN_MAJOR_VERSION 0x0003
```

Comment:
Could you please update this in src\tools\crossgen2\Common\Internal\Runtime\ModuleHeaders.cs as well?